### PR TITLE
ja-JP: Fill in some gaps in interface translation

### PR DESCRIPTION
**Pull request by AaronVanGeffen**
This uses strings from #1771 to fill in some of the gaps in the interface translation.

I wasn't quite satisfied with my own translations, so having the original game's strings for reference is a big help. However, I'm not quite happy with some of the choices there either.

Unifying both will be a task for another day. We can fill out some of the gaps, for now, however, with minor revisions.

### CODE REVIEW

File: data/language/ja-JP.txt [hunk 1]
```diff
@@ -105,7 +105,7 @@ STR_0519    :乗客はう特別にデザインされた車両に乗り、コー
 STR_0520    :A dock platform where guests can drive/row personal watercraft on a body of water
 STR_0521    :小さなネズミ型の車両がヘアピンコーナーと急な落下を横断して不安定に身を乗り出す。強烈度が高いはずです。
 STR_0522    :単線のコースタートラックに沿って走る、馬の形をした車両に乗客を乗せる。
-STR_0523    :Riders travel slowly in powered vehicles along a track-based route
+STR_0523    乗客は、軌道の付いたルートを動力付きの車両でゆっくりと移動します
```
Comment (Gymnasiast) [inline data/language/ja-JP.txt]: You accidentally removed the colon here.
```suggestion
STR_0523    :乗客は、軌道の付いたルートを動力付きの車両でゆっくりと移動します
```

Comment (AaronVanGeffen) [reply data/language/ja-JP.txt]: Ouch, thanks for catching that.

File: data/language/ja-JP.txt [hunk 2]
```diff
@@ -2282,14 +2287,14 @@ STR_3169    :Data for the following object not found:
...
 STR_3172    :The following object must be selected first: {STRING}
-STR_3173    :This object is currently in use
-STR_3174    :This object is required by another object
-STR_3175    :This object is always required
-STR_3176    :Unable to select this object
-STR_3177    :Unable to de-select this object
-STR_3178    :At least one path object must be selected
-STR_3179    :At least one ride vehicle/attraction object must be selected
-STR_3180    :Invalid selection of objects
+STR_3173    :{BLACK}(ID: {WINDOW_COLOUR_2}{BLACK}{COMMA16}のオブジェクトを含む{WINDOW_COLOUR_2}文字: {BLACK}{STRINGID}以下のオブジェクトデータが見つかりませんでした: 充分なグラフィックスペースがありませんこの種類のオブジェクトの選択が多すぎます以下のオブジェクトは最初に選択する必要があります: このオブジェクトは現在使用中です
```
Comment (AaronVanGeffen) [inline data/language/ja-JP.txt]: This is not the right string.

Comment (Gymnasiast) [reply data/language/ja-JP.txt]: Oops, merged it _just_ before you said that :/

Comment (AaronVanGeffen) [inline data/language/ja-JP.txt]:
```suggestion
STR_3173    :このオブジェクトは現在使用中です
```

Comment (AaronVanGeffen) [reply data/language/ja-JP.txt]: Ah, shoot. I'll pop in a quick fix on master, then.

**Pull request merged**